app/i18n/cz/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Dokončit instalaci',
 		'fix_errors_before' => 'Chyby prosím před přechodem na další krok opravte.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
Thanks. Git is too confusing for me. I'm running Funtoo Linux, I get the git commands, but... :-(